torch/distributed/rpc/__init__.py

```diff
@@ -78,7 +78,17 @@ def init_rpc(
         # and others might not have. As a result, a node calling
         # torch.distributed.autograd.backward() would run into errors since
         # other nodes might not have been initialized.
-        torch.distributed.autograd._init(rank)
+        dist_autograd._init(rank)
 
         # Initialize RPC.
         _init_rpc_backend(backend, store, name, rank, world_size, rpc_backend_options)
+
+
+    @_require_initialized
+    def _get_debug_info():
+        from . import _rref_context_get_debug_info
+        from .api import _agent
+        info = _rref_context_get_debug_info()
+        info.update(_agent.get_debug_info())
+        info.update(dist_autograd._get_debug_info())
+        return info
```

Comment:
Are we planning to keep this in Python or should we eventually refactor to have it all done in the C++ code?

Reply:
Good point! It should eventually live in C++, as we also want to have this API available for C++ use cases. For now, it's probably OK to have it here as we don't have public C++ APIs yet.